forch/heart_beat_scheduler.py

```diff
@@ -0,0 +1,28 @@
+"""Schedule heart beat with functions that have to be called in order"""
+import logging
+import threading
+
+LOGGER = logging.getLogger('HeartBeat')
+
+
+class HeartBeatScheduler:
```

Comment:
HeartbeatScheduler -- heartbeat is one word (same with filename -- heartbeat_scheduler.py)

Reply:
Done